src/unitxt/operators.py

```diff
@@ -213,6 +213,97 @@ def process(
         return flatten_dict(instance, parent_key=self.parent_key, sep=self.sep)
 
 
+class WholeInputFormatter(StreamInstanceOperator):
```

Comment:
Maybe call it -  ModelInputFormatter?  This is inlined in other places in the code, where we call the "whole" input "model input".  E.g. in augmenters.

Reply:
Yes, renamed.